xonsh/aliases.py

```diff
@@ -73,6 +73,19 @@ def source_bash(args, stdin=None):
     return
 
 
+def exec(args, stdin=None):
```

Comment:
It sounds reasonable to make it a wrapper around execx. I thought about the naming conflict but left it consistent because I wanted to hear your thoughts what it should be called and whether having the exec keyword taken will shadow the Python `exec()` builtin when using the shell as well.